app/assets/javascripts/app/views/generic/searchable_multiple_select.jst.eco

```diff
@@ -0,0 +1,33 @@
+<div class="dropdown-toggle" data-toggle="dropdown">
+  <input
+    class="searchableSelect-main form-control js-input<%= " #{ @attribute.class }" if @attribute.class %>"
+    placeholder="<%= @attribute.placeholder %>"
+    value="<%= @attribute.valueName %>"
+    autocomplete="off"
+    <%= @attribute.required %>
+   >
+  <div class="searchableSelect-autocomplete">
+    <span class="searchableSelect-autocomplete-invisible js-autocomplete-invisible"></span>
+    <span class="searchableSelect-autocomplete-visible js-autocomplete-visible"></span>
+  </div>
+  <% if !@attribute.ajax: %><%- @Icon('arrow-down', 'dropdown-arrow') %><% end %>
+  <div class="small loading icon"></div>
+</div>
+<div class="dropdown-menu dropdown-menu-left dropdown-menu--has-submenu dropdown-menu-multiple js-dropdown">
+  <ul class="js-optionsList" role="menu">
+    <%- @options %>
+  </ul>
+  <%- @submenus %>
+</div>
+<div class="items">
+  <% if @attribute.value: %>
+    <% for value in @attribute.value: %>
+      <div>
+        <span class='selected_item' onclick="$(this).closest('div').remove()">
```

Comment:
Please don't use inline JS. Move it to the controller that handles the template.